src/Arcus.WebApi.Unit/Correlation/CorrelationController.cs

```diff
@@ -10,13 +15,15 @@ public class CorrelationController : ControllerBase
         public const string Route = "correlation";
 
         private readonly HttpCorrelationInfo _correlationInfo;
+        private readonly DiagnosticContext _diagnosticContext;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="CorrelationController"/> class.
         /// </summary>
-        public CorrelationController(HttpCorrelationInfo correlationInfo)
+        public CorrelationController(HttpCorrelationInfo correlationInfo, DiagnosticContext diagnosticContext)
```

Comment:
Can be removed I think, no?

Reply:
Hmm, I added this to have a way to retrieve the enriched log properties from a Controller endpoint. So, we would still require this if we want to verify the properties.

Reply:
Ok, it's testing so 🤷‍♂